azure-iot-device/azure/iot/device/common/pipeline/pipeline_events_base.py

```diff
@@ -22,6 +25,7 @@ class PipelineEvent(object):
     :type name: str
     """
 
+    @abc.abstractmethod
```

Comment:
>abstractmethod [](start = 9, length = 14)

Is this abstract? It has an implementation...

Reply:
discussing in teams

---
In reply to: [293419085](https://github.com/Azure/azure-iot-sdk-python-preview/pull/106#discussion_r293419085) [](ancestors = 293419085)